components/navigation/NavBar.js

```diff
@@ -129,6 +129,16 @@ export default function NavBar({
     setOpen(null);
   }, [asPath]);
 
+  const LanguagePicker = (
```

Comment:
use a different name for `language picker` everywhere which follows naming conventions and also make it clear that is being used for mobile nav only

Reply:
@sambhavgupta0705 done!